docs/getting-started/comparison.md

```diff
@@ -4,11 +4,356 @@ description:
 nav: 2
 ---
 
-⚠️ This doc is still under construction. https://github.com/pmndrs/zustand/discussions/1033
+## Redux
 
-## Why zustand over react-redux?
+Zustand and Redux are state management libraries for React. Here are some
+differences between these two libraries.
 
-- Simple and un-opinionated
-- Makes hooks the primary means of consuming state
-- Doesn't wrap your app in context providers
-- [Can inform components transiently (without causing render)](recipes#transient-updates-for-often-occurring-state-changes)
+### State Model
+
+There are no big difference between Zustand and Redux. Both are base on
+immutable state model. Also, Redux needs to wrap you app in context providers.
+
+```ts
+import create from 'zustand'
+
+type State = {
+  count: number
+}
+
+type Actions = {
+  increment: (qty: number) => void
+  decrement: (qty: number) => void
+}
+
+const useCountStore = create<State & Actions>((set) => ({
+  count: 0,
+  increment: (qty: number) => set((state) => ({ count: state.count + qty })),
+  decrement: (qty: number) => set((state) => ({ count: state.count - qty })),
+}))
+```
+
+```ts
+import create from 'zustand'
+
+type State = {
+  count: number
+}
+
+type Actions = {
+  increment: (qty: number) => void
+  decrement: (qty: number) => void
+}
+
+type Action = {
+  type: keyof Actions
+  qty: number
+}
+
+const countReducer = (state: State, action: Action) => {
+  switch (action.type) {
+    case 'increment':
+      return { count: state.count + action.qty }
+    case 'decrement':
+      return { count: state.count - action.qty }
+    default:
+      return state
+  }
+}
+
+const useCountStore = create<State & Actions>((set) => ({
+  count: 0,
+  dispatch: (action: Action) => set((state) => countReducer(state, action)),
+}))
+```
+
+```ts
+import { createStore } from 'redux'
```

Comment:
This is traditional Redux. If you make a comparison note now, you want to compare with modern Redux.

Reply:
What you mean with modern Redux? Actually, Redux toolkit use createStore behind scenes. I can update the code to use the Redux Toolkit version but I want to show the basic example because that's why people often use even these days.

Reply:
I mean Redux Toolkit. The bare createStore should be marked as deprecated.

Reply:
I added a piece of code for Redux Toolkit as well